src/main/ruby/core/float.rb

```diff
@@ -127,7 +120,7 @@ def to_r
   def arg
     if nan?
       self
-    elsif signbit?
+    elsif self < 0 or equal?(-0.0)
```

Comment:
I don't know if we have a Rubocop rule for this, but I thought we settled on `||` for simple boolean expressions like this.